test.js

```diff
@@ -129,7 +139,7 @@ it('normal throw', function() {
     'throw new Error("test");'
   ], [
     'Error: test',
-    /^    at Object\.exports\.test \(.*\/line1\.js:1001:101\)$/
+    /^    at Object\.exports\.test \((?:.*\/)?line1\.js:1001:101\)$/
```

Comment:
The problem is that `compareStackTrace` runs the test twice, once with a source map in a file on disk and once with an inline source map.

One gives us results with a full path, the other used to give a `./` relative path and now gives a bare name.

The regular expressions need to match both however :-(